feeds/src/components/oracleTable/OracleTable.enhanced.ts

```diff
@@ -1,19 +1,18 @@
 import { connect } from 'react-redux'
 import OracleTable from './OracleTable.component'
 import {
-  aggregationSelectors,
-  aggregationOperations,
-} from 'state/ducks/aggregation'
+  aggregatorSelectors,
+  aggregatorOperations,
+} from 'state/ducks/aggregator'
 import { AppState } from 'state'
 
 const mapStateToProps = (state: AppState) => ({
-  networkGraphNodes: aggregationSelectors.networkGraphNodes(state),
-  networkGraphState: aggregationSelectors.networkGraphState(state),
-  ethGasPrice: state.aggregation.ethGasPrice,
+  ethGasPrice: state.aggregator.ethGasPrice,
+  latestOraclesState: aggregatorSelectors.latestOraclesState(state),
 })
 
 const mapDispatchToProps = {
-  fetchEthGasPrice: aggregationOperations.fetchEthGasPrice,
+  fetchEthGasPrice: aggregatorOperations.fetchEthGasPrice,
 }
 
 export default connect(mapStateToProps, mapDispatchToProps)(OracleTable)
```

Comment:
Can you combine `.component` with `.enhanced` e.g. https://github.com/smartcontractkit/chainlink/blob/develop/feeds/src/components/listing/Listing.tsx